modules/ROOT/pages/connected-apps-org-admin.adoc

```diff
@@ -2,37 +2,84 @@
+* loses administrative privileges over the organization.
+
+To maintain security, manually rotate the client secrets of all connected apps in the organization after the transfer. For more information about how to manually rotate client secrets, see <<modify-client-secret>>.
+
+You can also reset client secrets programmatically using the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/access-management-api/minor/1.0/console/method/%236918/[Access Management API]. 
+
+== Manage Owned Apps
 
 To maintain security, you can view and manage connected apps only if you have the Organization Administrator permission in the root organization or in the business group where the connected app resides.
 
-When an organization administrator leaves the organization, is deleted, or is disabled, ownership of their connected apps automatically transfers to the organization owner. To maintain security, manually rotate the client secrets of all connected apps in the organization after the transfer. For more information about how to rotate client secrets, see xref:connected-apps-developers.adoc#modify-the-client-secret-for-a-connected-app[Modify the Client Secret for a Connected App].
+To manage your connected apps:
 
-== Authorizations
+. Navigate to *Access Management* and select the *Connected Apps* tab. 
+. Add and edit apps in the *Owned Apps* section. 
+. Manage app access in the *Authorizations* section.
 
-You can use the *Authorizations* section to manage which apps are authorized to access your users' data. Click the *...* icon next to an application to view more details or to remove the app from your organization.
-When you revoke an app’s access, the app’s OAuth 2.0 access tokens and refresh tokens are revoked, and the app can no longer access user data.
+=== Edit the App Owner
 
-You can configure one of two types of authorization policies for your users:
+Organization administrators can edit the owner of the connected app and transfer the ownership to other administrators:
 
-* Unrestricted
+. In Access Management > *Connected Apps* > *Owned Apps*, click the app name. 
+. Make the changes and click *Save changes*.
 
-  Organization members can authorize any connected app to act on their behalf.
+[[modify-client-secret]]
+=== Modify the Client Secret for a Connected App
 
-* Restricted
+include::partial$include-nav-connected-apps.adoc[]
+
+. Click a connected app from the list to enable Edit mode.
+. In the *Secret* field, enter a secret of your choice.
+. Click *Save*.
+
+image::connected-apps-edit-mode.png[Edit mode for a connected app]
++
+[calloutlist]
+. The name and username of the owner of the connected app
+. The name of the connected app
+. The ID of the connected app
+. The client secret of the connected app
+
+=== View Authorizations
+
+. In the *Authorizations* tab, select the app.
+. Click the more actions menu (*...*) for the application and select *Show details*.
```

Comment:
good place for an icon image...

Reply:
Good idea. I'll add this icon later on. I think there are many occurrences of the ... in AM.